addons/mail/static/src/components/message/message.scss

```diff
@@ -239,6 +239,6 @@
 // Used to hide buttons on rating emails in chatter
 // FIXME: should use a better approach for not having such buttons
 // in chatter of such messages, but keep having them in emails.
-.o_Message_content [summary~="o_mail_notification"] {
+.o_Chatter .o_Message_content [summary~="o_mail_notification"] {
```

Comment:
Hello, this is not a correct way to do it, but I don't really understand the issue to suggest something better, and the task id leads nowhere so I can't really help.

Reply:
- First reason it's not ok because we don't want css specificity.
- But main reason is because it's not by restricting its display depending on its parent context that we'll resolve the issue, it will just create inconsistencies between the different views.

According to the report, what we want is to display it only if the message is targeted to the current user. So we should conditionally add a modifier class on `.o_Message_content` when it's for the current user, and then style accordingly.

Reply:
:astonished: :+1: